src/resources/OrganizationAccess/AccessInterfaces.ts

```diff
@@ -22,9 +22,32 @@ export interface AccessParams {
 }
 
 export interface AccessModel {
-    accessLevel: AccessLevel[];
-    callerPartOf: boolean;
+    /*
+     * The access level for this privilege.
+     */
+    accessLevel: AccessLevel;
+    /*
+     * Whether the calling user is part of this group.
+     */
+    callerPartOf?: boolean;
+    /*
+     * The creation date of this privilege.
+     */
+    createdDate?: string;
+    /*
+     * The display name of this privilege.
+     */
     displayName: string;
```

Comment:
This equals to the name of the group when privilegeHolderType is a group, and it equals to the name of the api key in the case where privilegeHolderType is an api key.